docs/faq.md

```diff
@@ -0,0 +1,71 @@
+
+6. How do I delete data?
+
+    Right now, there are two ways of deleting ingested data:
+        - Automatically using the data retention properties ([details here](https://github.com/timescale/promscale/blob/master/docs/sql_schema.md#data-retention))
+        - Manually through SQL interface (either Postgres [truncate table](https://www.postgresql.org/docs/current/sql-truncate.html) or [drop_chunks](https://docs.timescale.com/latest/api#drop_chunks) TimescaleDB function)
+
+7. What is the best way to set up HA (high availability) on Promscale?
+
+    You can find the documentation on Promscale HA setup [here](https://github.com/timescale/promscale/blob/master/docs/high-avaliability/prometheus-HA.md).
+
+8. What is this Promscale extension for?
+
+    Promscale extension is an additional extension separate from TimescaleDB that can be used to improve the general performance characteristics of Promscale. You can find more info here: https://github.com/timescale/promscale_extension
+
+9. Does Promscale support backfill?
+
+    Yes. Promscale support backfill of old data by sending older date to Promscale using the Prometheus [remote write API](https://prometheus.io/docs/prometheus/latest/storage/#remote-storage-integrations). This requires creating a custom application that can send requests to that API. We have plans in the near future to expose an SQL API to ingest data into Promscale.
+
+10. How do I store existing data from Prometheus to Promscale?
+
+    Right now, there is no good way to migrate existing data from your Prometheus instance to Promscale. We will be working on a tool which would enable you to do so in the near future.
+
+11. Why doesn’t Prometheus show Promscale data when I use the PromQL dashboard?
+
+    If you have backfilled data which is not ingested via Prometheus itself, you will need to enable the `read_recent` option for Promscale remote read endpoint ([details here](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_read)).
+
+12. How do I make the maximum use of Promscale in terms of efficient storage use? 
+
+    Since all your Prometheus data will be stored in Promscale as well, we suggest keeping the Prometheus retention period to 6 hours so your Prometheus instance will not take up unnecessary storage space.
+
+13. Are recording rules supported by Promscale? 
+
+    Yes. Prometheus manages the rules and Promscale will create the series on command from Prometheus.
+
+14. How can I find out more about Promscale?
+
+    If you have any questions, please join the #prometheus channel on [TimescaleDB Slack](https://slack.timescale.com/), or the [Promscale Users Google Group](https://groups.google.com/forum/#!forum/promscale-users). You can also join our users & community meeting ([details here](https://groups.google.com/forum/#!forum/promscale-users)).
```

Comment:
`You can also join our users & community meeting` but I don't think the provided link contains the link/details for **community meeting**.